tests/bson/bson-dbpointer-jsonserialize-001.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+MongoDB\BSON\Javascript::jsonSerialize() return value (without scope)
```

Comment:
I assume this was copy/pasted prematurely, since `DBPointer::jsonSerialize()` isn't implemented yet (although the other `jsonSerialize()` methods appear to be).